config/application.yml.default

```diff
@@ -96,13 +99,14 @@ liveness_checking_enabled: 'false'
 logins_per_ip_track_only_mode: 'false'
 # LexisNexis #####################################################
 lexisnexis_base_url: https://www.example.com
-lexisnexis_timeout: '10'
 lexisnexis_request_mode: testing
 # Instant Verify and Phone Finder Integrations
 lexisnexis_account_id: test_account
 lexisnexis_username: test_username
 lexisnexis_password: test_password
+lexisnexis_phone_finder_timeout: 1
 lexisnexis_phone_finder_workflow: customers.gsa.phonefinder.workflow
+lexisnexis_instant_verify_timeout: 1
```

Comment:
Should I expect to see a TrueID timeout configuration here?

Reply:
We already have `lexisnexis_trueid_timeout` config, or do you mean something separate?

Reply:
> We already have `lexisnexis_trueid_timeout` config, or do you mean something separate?

Ah! I missed that since it wasn't in the changes (I assumed `lexisnexis_timeout` was used for all LN implementations).

All good then 👍

Reply:
I also did some additional cleanup in 801b82962a68695c38f8ecf86100c55137856a49